src/components/application_manager/test/commands/hmi/simple_requests_to_hmi_test.cc

```diff
@@ -114,6 +140,9 @@ class RequestToHMICommandsTest
   typedef Command CommandType;
 };
 
+template <class Command>
```

Comment:
@VVeremjova typename instead of class according https://github.com/smartdevicelink/sdl_core/wiki/SDL-Coding-Style-Guide